### PR TITLE
Fix missing file for chinese

### PR DESCRIPTION
**Pull request by chuan9**
BUG=XWALK-6915

### CODE REVIEW

**Comment by chuan9** [comment]
@jondong @lisa0314 Please have a review.

File: public/documentation/android/android_target_setup_zh.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# 安卓机器配置
```
Comment (jondong) [inline public/documentation/android/android_target_setup_zh.md]: 我在想是不是叫安卓设备比较好一点？机器听起来怪怪的

**Comment by jondong** [comment]
@chuan9 nice translation, only a nit, thanks

**Comment by chuan9** [comment]
@jondong Done, thanks for your review.

**Comment by jondong** [comment]
LGTM, thanks

**Comment by baleboy** [comment]
Where are these new pages linked from? They are not in the sidebar and I couldn't find other pages referencing them. Also, the page/documentation/android_zh.html still refers to the untranslated page /documentation/android/android_target_setup.html

**Comment by baleboy** [comment]
The page /documentation/android_zh.html still refers to the untranslated page /documentation/android/android_target_setup.html, but that can be fixed in another PR. Merging.

**Pull request merged**